doap_Brooklyn.rdf

```diff
@@ -0,0 +1,77 @@
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+  <Project rdf:about="http://brooklyn.apache.org">
+    <created>2016-02-23</created>
+    <license rdf:resource="http://spdx.org/licenses/Apache-2.0" />
+    <name>Apache Brooklyn</name>
+    <homepage rdf:resource="http://brooklyn.apache.org" />
+    <asfext:pmc rdf:resource="https://svn.apache.org/repos/asf/comdev/projects.apache.org/data/committees/brooklyn.rdf" />
+    <shortdesc>Apache Brooklyn is a framework for modeling, monitoring, and managing applications through autonomic blueprints.</shortdesc>
+    <description>Apache Brooklyn is a framework for modeling, monitoring, and managing applications through autonomic blueprints.
+
+        Blueprints written in YAML describe your application and are composed from the dozens of supported components and
+        your own components using Bash, Java, Puppet, Chef etc.
+
+        Multi-tiered applications described in Blueprints can be deployed, configured and integrated across 20+ public
+        clouds (Amazon EC2, CloudStack, OpenStack, SoftLayer and many more), private clouds, Docker containers or bare
+        servers.
+
+        Once deployed Apache Brooklyn monitors key application metrics to provide dynamic policy management, including
+        autoscaling to meet demand, and resilisancy though replacing/restarting failed components.
+
+        Management of Apache Brooklyn is made possible via a dynamic web console, rich REST API and simple yet powerful
+        command line interface.</description>
+    <bug-database rdf:resource="https://issues.apache.org/jira/browse/BROOKLYN/" />
+    <mailing-list rdf:resource="http://brooklyn.apache.org/community/mailing-lists.html" />
+    <download-page rdf:resource="http://brooklyn.apache.org/download/index.html" />
+    <programming-language>Java</programming-language>
+    <category rdf:resource="http://projects.apache.org/category/cloud" />
+    <release>
+      <Version>
+        <name>Brooklyn 0.8.0-incubating</name>
+        <created>2015-09-14</created>
+        <revision>0.8.0-incubating</revision>
+      </Version>
+    </release>
+    <release>
+      <Version>
+        <name>Brooklyn 0.7.0-incubating</name>
+        <created>2015-07-15</created>
+        <revision>0.7.0-incubating</revision>
+      </Version>
+    </release>
+    <repository>
+      <GitRepository>
+        <location rdf:resource="https://git-wip-us.apache.org/repos/asf/brooklyn.git"/>
+        <browse rdf:resource="https://git-wip-us.apache.org/repos/asf?p=brooklyn.git"/>
+      </GitRepository>
+    </repository>
+    <maintainer>
+      <foaf:Person>
+        <foaf:name>Richard Downer</foaf:name>
+          <foaf:mbox rdf:resource="mailto:richard.downer@cloudsoftcorp.com "/>
```

Comment:
Please use "richard@apache.org" - thanks!

Can we specify multiple maintainers? If so then please add all the Brooklyn committers:

```
    Aled Sage                  <aledsage@apache.org>            [2015-11-18]
    Alex Heneveld              <heneveld@apache.org>            [2015-11-18]
    Andrea Turli               <andreaturli@apache.org>         [2015-11-18]
    Andrew Kennedy             <grkvlt@apache.org>              [2015-11-18]
    Ciprian Ciubotariu         <cipi@apache.org>                [2015-11-18]
    Hadrian Zbarcea            <hadrian@apache.org>             [2015-11-18]
    Jean-Baptiste Onofré       <jbonofre@apache.org>            [2015-11-18]
    Olivier Lamy               <olamy@apache.org>               [2015-11-18]
    Richard Downer (chair)     <richard@apache.org>             [2015-11-18]
    Sam Corbett                <sjcorbett@apache.org>           [2015-11-18]
    Svetoslav Neykov           <svet@apache.org>                [2015-11-18]
```